packages/core/src/tracing/instrumentation/database/couchbase.js

```diff
@@ -117,28 +127,39 @@ function instrumentCluster(cluster, connectionStr) {
     };
   });
 
-  // #### ANALYTICS SERVICE
-  shimmer.wrap(cluster, 'analyticsQuery', function instanaClusterAnalyticsQuery(original) {
-    return function instanaClusterAnalyticsQueryWrapped() {
-      const originalThis = this;
-      const originalArgs = arguments;
-      const sqlStatement = originalArgs[0] || '';
-
-      return instrumentOperation(
-        {
-          connectionStr,
-          sql: tracingUtil.shortenDatabaseStatement(sqlStatement),
-          resultHandler: (span, result) => {
-            if (result && result.rows && result.rows.length > 0 && result.rows[0].BucketName) {
-              span.data.couchbase.bucket = result.rows[0].BucketName;
-              span.data.couchbase.type = bucketLookup[span.data.couchbase.bucket];
+  // v4.4.4 introduced a breaking code change.
+  // We are no longer able to extract the SQL statements for "analyticsindexes".
+  // see chnages: https://github.com/couchbase/couchnode/compare/v4.4.3...v4.4.4?diff=split&w=
```

Comment:
```suggestion
  // see changes: https://github.com/couchbase/couchnode/commit/b8118b8dd05c710e0f0d898ba8e16372028ea294
```
a typo and points to the exact commit